packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -796,7 +790,7 @@ export class Markets {
       ).dividedBy(QUINTILLION);
       const settlementFee = marketCreatorFeeRate.plus(reportingFeeRate);
       // TODO: find this value from logs.
-      const noShowBondAmount = "999999000000000000000";
+      const noShowBondAmount = '999999000000000000000';
```

Comment:
was this added to market created log?

Reply:
looks like no:
`packages/augur-core/source/contracts/Augur.sol:46`:

```    event MarketCreated(IUniverse indexed universe, uint256 endTime, string extraInfo, IMarket market, address indexed marketCreator, address designatedReporter, uint256 feePerCashInAttoCash, int256[] prices, IMarket.MarketType marketType, uint256 numTicks, bytes32[] outcomes, uint256 timestamp);
```

Reply:
None of the other logs defined in Augur.sol have it either.

Reply:
thanks